web/src/app/modules/views/main/editors/modules/contents-container/services/content-container.service.ts

```diff
@@ -0,0 +1,18 @@
+import { EventEmitter, Injectable } from '@angular/core';
+import { IContainer } from '../../../../../../../model/IContainer';
+import { Subject } from 'rxjs';
+
+ @Injectable()
+export class ContentsContainerService {
+
+   private onModelDeleted = new Subject<string>();

```

Comment:
Why is the Subject typed to string? We do not pass any data, thus it should be void.